### ISSUE TITLE
Make compatible with JSON Resume

### ISSUE THREAD

**Issue by victorperin** [salomonelli/best-resume-ever#3]
Guys I just know about this project, and it's awesome! :)

Right now I know about [JSON Resume](https://jsonresume.org/), it's a compatible format of resumes.
Could this project someday be compatible with that?

**Comment by salomonelli** [comment]
Hello @victorperin 

that is a great idea. I didn't know, that JSON Resume exists. Would you mind making a PR? That would be really great.

**Comment by czonios** [comment]
Hey @salomonelli 

JSON Resume doesn't look that much different in format than your /src/person.js file.
I think that you could fetch the data from a JSON file in a way very similar to how you are doing now with person.js. Basically:

  * change **template.vue** to import  from the JSON file
  * change  each template to use e.g. `{{basics.name}}` instead of `{{person.name.first}}  {{person.name.last}}` 
  
Please note that this approach will make /src/person.js obsolete.
I have used JSON before but only with jQuery. I can work on this if you want to assign it to me, or preferably I can work together with someone else.

**Comment by pubkey** [comment]
I see some problems with json-resume.

- One is that is does not split pre- and surname and likely other things would have to be monkey-patched.
- The bigger one is that the last commit is from feb2017 and it looks like none of the issues is adressed.

I think a standard for resume-data-format would be nice but I don't see a good reason to switch to json-resume.

**Comment by victorperin** [comment]
@pubkey I view json-resume only as a JSON format. The format could bring json-resume users and help this project even more. :)

**Comment by Dannyzen** [comment]
Agreed with @victorperin on this. I've been a user of json-resume, [hackmyresume](https://github.com/hacksalot/HackMyResume) and [fresh](https://github.com/fresh-standard/) and am looking for a new rendering home for my resume as the other projects have started to die off. Having best-resume-ever be that project would be very exciting and valuable.

**Comment by salomonelli** [comment]
The format of person.js is specifically adjusted to fit the templates and is meant to be flexible. 

- As @pubkey  already mentioned, a separation of first and second names is required for some templates and can not be achieved through splitting strings (double names, foreign names, etc.). 
- Some users would like to add bullet points to descriptions (see #17 ). With json-resume they would need to write markdown or html into their json file instead of just adjusting the object.

A standardized format is great. But users should be able to create any kind of template easily without being restricted to the json-resume-format.

**Comment by czonios** [comment]
@pubkey and @salomonelli

Please review the second part of my post, I think it resolves all those issues while making use of the more widely accepted json resume format.

**Comment by Dannyzen** [comment]
@pubkey @salomonelli perhaps the question that should be asked is: Is the goal of this project to obtain more users quickly? If it is, supporting pre-existing formats might make sense. If it's not, then those of us looking for an alternative who are married to our current format need to wait for support or look elsewhere. 

Hope this approach to feedback changes the conversation from technical implementation to project vision :)

**Comment by nic611** [comment]
It would be great if a template in Chinese (UI and lang) included !

**Comment by kaufmann42** [comment]
So what was the verdict on all of this? Honestly, it's my opinion that JSON Resume is a very cool project with a lot of great ideas. It's a shame that it went by the wayside as the maintainers slowly moved away from it.

To take advantage of some of the things they got right and apply them here would most definitely be a boon.
